mdsuite/database/simulation_database.py

```diff
@@ -97,6 +97,23 @@ def __eq__(self, other):
         return same
 
 
+@dataclasses.dataclass
+class MoleculeInfo(SpeciesInfo):
+    """Information about a Molecule
+
+    All the information of a species + groups
+
+    Attributes
+    ----------
+    groups: dict
+        A molecule specific dictionary for mapping the molecule to the
+        particles. TODO SamTov can you explain a bit more because I haven't found
+        a good docstring to copy/paste from map_molecules.py
```

Comment:
@SamTov could you update the docstrings here?

Reply:
+ update notebook

Reply:
Resolved in [1aaa880](https://github.com/zincware/MDSuite/pull/497/commits/1aaa880c3a1bdeff4e715b2912d089de64877d33)